### PR TITLE
Include README.md in System.Drawing.Common package building.

### PR DESCRIPTION
**Pull request by KlausLoeffelmann**
The README in `System.Drawing.Common` was never included in the package building.
This PR fixes that.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/9844)

### CODE REVIEW

File: src/System.Drawing.Common/README.md [hunk 1]
```diff
@@ -8,22 +8,6 @@ To get started with `System.Drawing.Common`, install it using the NuGet Package
...
-
-### .NET CLI
-```
-dotnet add package System.Drawing.Common --version [your_version_here]
-```
-
-### Prerequisites
-
-- .NET Core 2.0+ (min .NET Core 3.1 for WinForms/WPF Apps)
-- .NET 5+
```
Comment (JeremyKuhne) [inline src/System.Drawing.Common/README.md:24]: There is a reference to 5+ earlier in the file as well, should be 6+.

**Comment by ViktorHofer** [comment]
What I mentioned [here](https://github.com/dotnet/windowsdesktop/pull/3835#issuecomment-1703765361) is that you want to have a PACKAGE.md file with the corresponding package authoring changes that link to it and a separate README.md file for repository source code contributors.

File: src/System.Drawing.Common/README.md [hunk 2]
```diff
@@ -1,72 +1,16 @@
...
-- [Drawing with System.Drawing.Common | Microsoft Learn](https://learn.microsoft.com/dotnet/core/drawing/)
+Documentation can be found at https://learn.microsoft.com/en-us/dotnet/api/system.drawing.
 
 ## Contribution Bar
-- [x] We consider new features, new APIs and performance changes
-- [x] We consider PRs that target this library for new source code analyzers
-- [ ] We don't accept refactoring changes due to new language features
-
-See the [Help Wanted](https://github.com/dotnet/winforms/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-System.Drawing+label%3A%22help+wanted%22) issues.
+- [x] [We consider new features, new APIs and performance changes](../README.md#primary-bar)
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/README.md:9]: These links are still incorrect

File: src/System.Drawing.Common/README.md [hunk 3]
```diff
@@ -1,72 +1,16 @@
...
+- [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [ ] [We don't accept refactoring changes due to new language features](../README.md#secondary-bars)
 
-## Feedback
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aissue+is%3Aopen+label%3Aarea-System.Drawing+label%3A%22help+wanted%22) issues.
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/README.md:13]: This link should point to winforms

File: src/System.Drawing.Common/README.md [hunk 4]
```diff
@@ -1,72 +1,16 @@
 # System.Drawing.Common
+This assembly provides access to GDI+ basic graphics functionality via types such as [`Bitmap`](https://learn.microsoft.com/en-us/dotnet/api/system.drawing.bitmap) and [`Font`](https://learn.microsoft.com/en-us/dotnet/api/system.drawing.font).
 
-The `System.Drawing.Common` package allows .NET Core and .NET 5+ applications to access GDI+ graphics functionality. This package is especially useful for porting .NET Framework applications that rely on the `System.Drawing` namespace.
+Note that `System.Drawing.Common` is only supported on Windows: https://learn.microsoft.com/en-us/dotnet/core/compatibility/core-libraries/6.0/system-drawing-common-windows-only.
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/README.md:4]: Please remove en-us

File: src/System.Drawing.Common/src/package.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
+# System.Drawing.Common
+
+The `System.Drawing.Common` package allows .NET Core and .NET 5+ applications to access GDI+ graphics functionality. This package is especially useful for porting .NET Framework applications that rely on the `System.Drawing` namespace.
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/src/package.md]: 5->6

File: src/System.Drawing.Common/src/package.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+For more in-depth tutorials and API references, you can check the following resources:
+
+- [NuGet Gallery | System.Drawing.Common](https://nuget.org/packages/System.Drawing.Common/)
+- [System.Drawing.Common Namespace | Microsoft Docs](https://docs.microsoft.com/dotnet/api/system.drawing)
+- [Drawing with System.Drawing.Common | Microsoft Learn](https://learn.microsoft.com/dotnet/core/drawing/)
+
+## Feedback
+
+- Open an issue on the [GitHub repository](https://github.com/dotnet/winforms/issues)
+- Reach out on Twitter with the hashtag #winforms
```
Comment (Tanya-Solyanik) [inline src/System.Drawing.Common/src/package.md:65]: Do you want to add a link for twitter?

File: src/System.Drawing.Common/README.md (hunk 2 above)
Comment (KlausLoeffelmann) [reply src/System.Drawing.Common/README.md:9]: Ah damn, I just restored the old README. 🤦

**Review by Tanya-Solyanik — APPROVED** [review]

**Review by ViktorHofer — APPROVED** [review]

**Pull request merged**